s3torchconnector/src/s3torchconnector/__init__.py

```diff
@@ -18,4 +19,5 @@
     "S3Reader",
     "S3Writer",
     "S3Exception",
+    "__version__",
```

Comment:
Would it make sense to export version with unique name to prevent possible confusions, because of names clashing?

Reply:
Not really, `__version__` is the PEP8 standard for naming this: https://peps.python.org/pep-0008/#module-level-dunder-names

If we need to import them both as part of a file we can have import aliases:
```
from s3torchconnector import __version__ as connector_version
from s3torchconnectorclient import  __version__ as client_version
```